src/main/java/com/robinpowered/sdk/model/Invitable.java

```diff
@@ -0,0 +1,7 @@
+package com.robinpowered.sdk.model;
+
+/**
+ * Interface for models that are invited to an {@link Event}.
```

Comment:
nitpicky, but since this is public "Interface for models that may be invited to an {@link Event}.